test/integration/targets/aws_acm/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+- name: AWS ACM integration test virtualenv wrapper
+  block:
+  - set_fact:
+      virtualenv: "{{ remote_tmp_dir }}/virtualenv"
+      virtualenv_command: "{{ ansible_python_interpreter }} -m virtualenv"
+  
+  - set_fact:
+      virtualenv_interpreter: "{{ virtualenv }}/bin/python"
+  
+  - pip:
+      name: virtualenv
+  
+  - pip:
+      name:
```

Comment:
What do you mean by "base image"?

The dependencies I need are a few lines down.

```
        - 'botocore<1.13.0,>=1.12.211'
        - boto3
        - coverage
        - jinja2
        - pyyaml
        - 'pyopenssl>=0.15'
        - 'cryptography>=1.6'
```

Reply:
I originally used `setup_openssl`. As per [this thread](https://github.com/ansible/ansible/pull/60552#discussion_r314266062) with @willthames , we decided to switch to the virtual env approach instead, to avoid 

> ERROR: Test target "aws_acm" has changed the test environment!

Reply:
Thank you @matt-telstra for the pointer.